### PR TITLE
[TACHYON-1040] Update TachyonFSTestUtils to use ClientOptions

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-1040

### CODE REVIEW

**Comment by miasantreble** [comment]
shall we disable this test while waiting for TACHYON-1043 to fixed?

**Comment by miasantreble** [comment]
jenkins, test this please.

**Comment by miasantreble** [comment]
jenkins, test this please.

**Comment by miasantreble** [comment]
@calvinjia @apc999 
Just resolved merge conflicts, can you please take a look if possible? Thanks!

File: clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java [hunk 1]
```diff
@@ -77,11 +76,25 @@ public static TachyonFile createByteFile(TachyonFileSystem tfs, String fileName,
...
+   *
+   * @param tfs a TachyonFileSystem handler
+   * @param fileURI URI of the file
+   * @param options client options to create the file with
+   * @param len file size
+   * @return the TachyonFile of the created file
+   * @throws IOException if <code>path</code> is invalid (e.g., illegal URI)
+   */
+  public static TachyonFile createByteFile(TachyonFileSystem tfs, TachyonURI fileURI,
+      OutStreamOptions options, int len) throws IOException {
```
Comment (uronce-cc) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: Why not just keep this method?

Comment (miasantreble) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: there were two implementations of the same method, one takes a string `fileName` and the other takes a TachyonURI. The difference between the two methods were non significant. I don't see why we want to keep them separate. Isn't it more maintainable to have only one impl and have everyone else calling it?

Comment (apc999) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: agreed

**Comment by apc999** [comment]
+1 LGTM

**Comment by uronce-cc** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**